components/button/button-subtle.js

```diff
@@ -59,19 +65,41 @@ class ButtonSubtle extends ButtonMixin(RtlMixin(LitElement)) {
 				}
 
 				button {
+					--d2l-button-subtle-padding-left: 0.6rem;
+					--d2l-button-subtle-padding-right: 0.6rem;
 					background-color: transparent;
 					border-color: transparent;
 					font-family: inherit;
-					padding: 0.55rem 0.6rem;
+					padding: 0 var(--d2l-button-subtle-padding-right) 0 var(--d2l-button-subtle-padding-left);
```

Comment:
Removing the `0.55rem` top and bottom padding is what fixes the problem with the regular subtle buttons being slightly larger than `42px`.  The `min-width` of `calc(2rem + 2px)` on all buttons is enough to determine the size, and the padding here made it a tad too big.  I can't find a case where the `padding` was needed... maybe old browsers?  Running visual-diff to see what it finds, also haven't tested Safari yet